From fd752f6141109bcdfe0dacaf5a8b168ac309c87a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:56 -0400
Subject: [redhat] kabi: Adding symbol kmalloc_order_trace (mm/slub.c)

Message-id: <dc68ff1c9ab2b0afadbf4240e68211599d5868ea.1410510773.git.poros@redhat.com>
Patchwork-id: 96107
O-Subject: [PATCH RHEL7.1 116/233] kabi: Adding symbol kmalloc_order_trace (mm/slub.c)
Bugzilla: 1077335
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Not directly used. Seems to be for debugging kmalloc part.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/kmalloc_order_trace
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/kmalloc_order_trace
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/kmalloc_order_trace

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/kmalloc_order_trace b/redhat/kabi/kabi-module/kabi_ppc64/kmalloc_order_trace
new file mode 100644
index 0000000..a97c34e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/kmalloc_order_trace
@@ -0,0 +1,2 @@
+#1-
+0xa202a8e5 kmalloc_order_trace vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/kmalloc_order_trace b/redhat/kabi/kabi-module/kabi_s390x/kmalloc_order_trace
new file mode 100644
index 0000000..7598320
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/kmalloc_order_trace
@@ -0,0 +1,2 @@
+#1-
+0x4b5814ef kmalloc_order_trace vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/kmalloc_order_trace b/redhat/kabi/kabi-module/kabi_x86_64/kmalloc_order_trace
new file mode 100644
index 0000000..a97c34e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/kmalloc_order_trace
@@ -0,0 +1,2 @@
+#1-
+0xa202a8e5 kmalloc_order_trace vmlinux EXPORT_SYMBOL
-- 
1.7.1